src/main/java/com/notnoop/apns/internal/SSLContextBuilder.java

```diff
@@ -0,0 +1,150 @@
+package com.notnoop.apns.internal;
```

Comment:
Fixed. I noticed some more files with license issues, so i added the mycila license check plugin. I had to reformat the licenses a bit to get the plugin to work with the default settings, but the licenses itself are not changed. By default you get the license with your name in it, but you can opt to remove the copyright header for files with multiple authors and it won't complain.